README.md

```diff
@@ -1,60 +1,30 @@
-# What is FAST-DNA?
-A design architecture for the web that enables real-time updating of user experiences based on changing inputs. The acronym stands for fast, adaptive, secure, timeless as a design network architecture.
 
-The majority of the effort in building component frameworks goes into the markup, accessibility, localization, and more. Let FAST-DNA do the hard work so you can focus on your design language and user experience.
+# What is FAST-DNA
+FAST-DNA is an acronym, which stands for Fast, Adaptive, Secure, and Timeless Design Network Architecture. A set of resources that can be used to accelerate building Web sites and applications.
 
+Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidance and related technologies. FAST-DNA solves this problem by providing resources to accelerate development with code that is performant and accessible. The resources work with multiple component frameworks such as (Fabric, MWF, Foundation, etc.) as well as JavaScript frameworks and libraries (React, Angular, Vue, etc.).
 
-## Packages
-[Lerna](https://github.com/lerna/lerna) in [independent mode](https://github.com/lerna/lerna#independent-mode---independent) is used to organize and manage the monorepository and contains a set of utilities each of which are published to NPM. Users can choose one package, some packages, or all of them together as an ACCELERATOR to quickly construct enterprise experiences. Highly tuned production code where the hard work has been done already.
-
-
-### React packages
-Built on ReactJS, bundled with Webpack, including Redux for state management.
-
-#### FAST components base
-A set of base components written in React that deliver quality HTML markup optimized for performance, accessibility, and modern browsers. A large amount of effort in all component frameworks often goes into just this, so delivering these base components as a package that then allows users to add a design language can reap major efficiency benefits in the design process. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-components-react-base)
-
-#### FAST components styled with for Microsoft
-A set of React components with a Microsoft design system driving the look and feel. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-components-react-msft)
-
-#### FAST development site
-A development site which can be used to create a React environment for a group of React or web components. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-development-site-react)
-
-#### FAST JSS manager
-A React higher-order component (HOC) library for managing component JSS stylesheets. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-jss-manager-react)
+**How do we accelerate?** A team can use these resources as guidance, reference, fork, or as dependencies.
```

Comment:
"fork" feels weird here - is there a better way we can say it? Maybe "starting point" or "base"?

Reply:
This feels ok to me because you can fork a project on github or choose to consume from NPM as a dependency.